apphub/automl/rand_augment/rand_augment_tf.py

```diff
@@ -0,0 +1,310 @@
+        displacement = int(self.level / 10 * height / 3 * random.choice([1.0, -1.0]))
+        im = im.transform((width, height),
+                          ImageTransform.AffineTransform((1.0, 0.0, 0.0, 0.0, 1.0, displacement)),
+                          resample=Image.BICUBIC)
+        return np.copy(np.asarray(im))
+
+
+def get_estimator(level,
+                  num_augment,
+                  epochs=24,
+                  batch_size=512,
+                  max_train_steps_per_epoch=None,
+                  max_eval_steps_per_epoch=None):
+    assert level <= 10, "the level should not be more than 10"
+    train_data, test_data = load_data()
+    aug_ops = [
+        OneOf(
+            Rotate(level=level, inputs="x", outputs="x", mode="train"),
+            Identity(level=level, inputs="x", outputs="x", mode="train"),
+            AutoContrast(level=level, inputs="x", outputs="x", mode="train"),
+            Equalize(level=level, inputs="x", outputs="x", mode="train"),
+            Posterize(level=level, inputs="x", outputs="x", mode="train"),
+            Solarize(level=level, inputs="x", outputs="x", mode="train"),
+            Sharpness(level=level, inputs="x", outputs="x", mode="train"),
+            Contrast(level=level, inputs="x", outputs="x", mode="train"),
+            Color(level=level, inputs="x", outputs="x", mode="train"),
+            Brightness(level=level, inputs="x", outputs="x", mode="train"),
+            ShearX(level=level, inputs="x", outputs="x", mode="train"),
+            ShearY(level=level, inputs="x", outputs="x", mode="train"),
+            TranslateX(level=level, inputs="x", outputs="x", mode="train"),
+            TranslateY(level=level, inputs="x", outputs="x", mode="train"),
+        ) for _ in range(num_augment)
+    ]
+    pipeline = fe.Pipeline(
+        train_data=train_data,
+        test_data=test_data,
+        batch_size=batch_size,
+        ops=aug_ops + [
+            Normalize(inputs="x", outputs="x", mean=(0.4914, 0.4822, 0.4465), std=(0.2471, 0.2435, 0.2616)),
+        ])
+    model = fe.build(model_fn=my_model, optimizer_fn="adam")
+    network = fe.Network(ops=[
+        ModelOp(model=model, inputs="x", outputs="y_pred"),
+        CrossEntropy(inputs=("y_pred", "y"), outputs="ce"),
+        UpdateOp(model=model, loss_name="ce")
+    ])
+    estimator = fe.Estimator(pipeline=pipeline,
+                             network=network,
+                             epochs=epochs,
+                             traces=Accuracy(true_key="y", pred_key="y_pred"),
+                             max_train_steps_per_epoch=max_train_steps_per_epoch,
+                             max_eval_steps_per_epoch=max_eval_steps_per_epoch)
+    return estimator
+
+
+if __name__ == "__main__":
+    result = np.zeros(shape=(10, 10))
+    for level in range(1, 11):
+        for num_augment in range(1, 11):
+            est = get_estimator(level=level, num_augment=num_augment, epochs=50)
+            est.fit(summary="exp")
```

Comment:
you can skip the summary here I think

Reply:
no, I tried it doesn't store the history object without summary

Reply:
if you put summary on .test() that should be enough to turn on history